src/mg_ore.cpp

```diff
@@ -20,9 +20,10 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "mg_ore.h"
 #include "mapgen.h"
 #include "noise.h"
-#include "util/numeric.h"
 #include "map.h"
 #include "log.h"
+#include <algorithm>
```

Comment:
`CONTAINS` requires it.  It's not included by `numerc.h` because only a few files actually use it and we don't want to require the compiler to load it for avarything that includes `numeric.h`.